atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/AdjustableSweepBatchConfigSource.java

```diff
@@ -87,7 +87,7 @@ public void decreaseMultiplier() {
         // Cut batch size in half, always sweep at least one row.
         reduceBatchSizeMultiplier();
 
-        log.warn("Sweep failed unexpectedly with candidate batch size {},"
+        log.info("Sweep failed unexpectedly with candidate batch size {},"
```

Comment:
probably outside the scope of this PR, but there is a case for a higher log level (WARN I think) if all params are 1.